src/platform/utilities/feature-toggles/helpers.js

```diff
@@ -0,0 +1,27 @@
+const Cookies = require('js-cookie');
+
+// Reads Google Analytics ID if available. Otherwise use a random string as a token.
+export function createTokenFromCookie(cookieName) {
+  return Cookies.get(cookieName)
+    ? Cookies.get(cookieName)
```

Comment:
Does this duplicate what we're doing in `setCookie`?

Reply:
Talked with Jhonny about this. This is for a totally different cookie. :sweat_smile: